frontend/src/components/OperationsTableCard/OperationsTableCard.tsx

```diff
@@ -57,17 +58,25 @@ const getSourceDatasetsLink = (
   }${sourceID}?limit=${limit}&offset=${offset}&search=${search}`;
 
 const OperationsTableCard: FunctionComponent<OperationsTableCardProps> = (props) => {
+  const { search } = useLocation();
+  const queryParams = queryString.parse(location.search);
```

Comment:
instead of `location.search`, use the `search` provided in the previous line... in fact, `useLocation()` eliminates the need for `window.location` so clear it out anywhere else.